doc/source/features.rst

```diff
@@ -48,18 +48,22 @@ only to a morphology or a morphology population.
 
 An example for ``neurite``:
 
-.. code-block:: python
+.. testcode::
 
-   from neurom import load_morphology, features
-   from neurom.features.neurite import max_radial_distance
+    from neurom import load_morphology, features
+    from neurom.features.neurite import max_radial_distance
+
+    m = load_morphology("tests/data/swc/Neuron.swc")
+
+    # valid input
+    rd = max_radial_distance(m.neurites[0])
+
+    # invalid input
+    # rd = max_radial_distance(m)
```

Comment:
As far as I can see [here](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testoutput) and [here](https://stackoverflow.com/a/12609/3868743), it is possible to check and show the exception that is raised by the invalid input.

Reply:
I would need to separate this line into a different snippet to capture that cleanly, and I don't think it is worth it tbh.

Reply:
I can understand, as you prefer 😁

Reply:
Let's leave it as it is for the time being. :D